stream/excel_stream_test.go

```diff
@@ -0,0 +1,418 @@
+package excel_stream
```

Comment:
Oh, I meant to change it. I actually never realized that the package name in Go doesn't have to match the folder name.